spec/features/posts_index_spec.rb

```diff
@@ -0,0 +1,48 @@
+# spec/features/user_show_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe 'User Show Page', type: :feature do
+  let!(:user) { User.create(name: 'Ajrass', photo: 'https://avatars.githubusercontent.com/u/130588108?v=4') }
+  let!(:post) { Post.create(author_id: user.id, title: 'Test Post', text: 'Hello world!') }
+  let!(:comment1) { Comment.create(user: user, post: post, text: 'Hello reviewer!') }
+  let!(:comment2) { Comment.create(user: user, post: post, text: 'Hope everything is just fine for you!') }
+
+  before { visit user_posts_path(user) }
+
+  it "displays the user's profile picture" do
+    expect(page).to have_css("img[src*='#{user.photo}']")
+  end
+
+  it 'displays the user\'s username' do
+    expect(page).to have_content(user.name)
+  end
+
+  it 'displays the number of posts the user has written' do
+    expect(page).to have_content(user.posts_counter)
+  end
+
+  it 'displays a post\'s title' do
+    expect(page).to have_content(post.title)
+  end
+
+  it 'displays some of the post\'s body' do
+    expect(page).to have_content(post.text)
+  end
+
+  scenario 'Displays the first comments on a post and total comments' do
+    expect(page).to have_content(comment2.text)
+    expect(page).to have_content("Comments: #{post.comments_counter}")
+  end
```

Comment:
- [ ] As required by the previous reviewer, As per [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20user%27s%20first%203%20posts.), you need to test that asserts the user's first comments, instead of testing only one comment, expand your test to include assertions for multiple comments. This will ensure thorough testing of the expected behavior, covering a broader range of scenarios.